components/AuthorCardList.tsx

```diff
@@ -16,6 +16,7 @@ const AuthorCardList: React.FunctionComponent<Props> = props => (
                 <AuthorCard
                     name={author.name}
                     bio={author.bio}
+                    id={author.id}
```

Comment:
Was this intended to be in the authors PR, instead? In any case, it seems fine to include it here, looks good.

Reply:
It was! The branch has been deleted after it was merged to main, so I thought I could just include it here.